dotnet-grad/Controllers/UserController.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using dotnet_grad.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace dotnet_grad.Controllers
+{
+  [ApiController]
+  [Route("api/[controller]")]
+  public class UserController : ControllerBase
+  {
+    TestContext testContext;
```

Comment:
this member is not being accessed outside of the current class or modified within it, in this case it should have a `private readonly` accessor and `private` members have the underscore notation.

Reply:
`readonly` can only be applied to members that are initialised in the constructor unless they have a default value of a getter of some sorts.